e2e/apprunner/copilot/back-end/manifest.yml

```diff
@@ -0,0 +1,56 @@
+# The manifest for the "back-end" service.
+# Read the full specification for the "Request-Driven Web Service" type at:
+# https://aws.github.io/copilot-cli/docs/manifest/rd-web-service/
+
+# Your service name will be used in naming your resources like log groups, App Runner services, etc.
+name: back-end
+# The "architecture" of the service you're running.
+type: Request-Driven Web Service
+
+image:
+  # Docker build arguments.
+  # For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/rd-web-service/#image-build
+  build: back-end/Dockerfile
+  # Port exposed through your container to route traffic to it.
+  port: 80
+
+http:
+  private: true
+  # healthcheck:
+  #   path: /
+  #   healthy_threshold: 3
+  #   unhealthy_threshold: 5
+  #   interval: 10s
+  #   timeout: 5s
+
+# Number of CPU units for the task.
+cpu: 1024
+# Amount of memory in MiB used by the task.
+memory: 2048
+
+# Connect your App Runner service to your environment's VPC.
+network:
+  vpc:
+    placement: private
+
+
+# Enable tracing for the service.
+# observability:
+#   tracing: awsxray
+
+# Optional fields for more advanced use-cases.
+#
+# variables:                    # Pass environment variables as key value pairs.
+#   LOG_LEVEL: info
+#
+# tags:                         # Pass tags as key value pairs.
+#   project: project-name
+#
+# secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store and Secrets Manager.
+#   GITHUB_TOKEN: GITHUB_TOKEN  # The key is the name of the environment variable, the value is the name of an SSM parameter.
+
```

Comment:
why do we create the manifest file without the secrets enabled? if it is not enabled how do we know if this feature works from e2e?

Reply:
I might have missed it. I'll send out an entirely new PR.